### ISSUE TITLE
Email sent to self can't be opened

### ISSUE THREAD

**Issue by kaeff** [pixelated/pixelated-user-agent#463]
1) Start version 035f3ce locally
2) Login to an account on wazokazi
3) sent mail to yourself
4) open mail

what should happen: I can read the mail in the details view
what happens: "Problem talking to sever" and the following stacktrace on the service's logs:

```
2015-09-18 14:49:14 [twisted] CRITICAL Unhandled error in Deferred:


Traceback (most recent call last):
  File "/Users/kfleerko/.virtualenvs/pixelated-user-agent/lib/python2.7/site-packages/twisted/internet/defer.py", line 1274, in unwindGenerator
    return _inlineCallbacks(None, gen, Deferred())
  File "/Users/kfleerko/.virtualenvs/pixelated-user-agent/lib/python2.7/site-packages/twisted/internet/defer.py", line 1128, in _inlineCallbacks
    result = g.send(result)
  File "/Users/kfleerko/code/gh/pixelated/pixelated-user-agent/service/pixelated/adapter/mailstore/leap_mailstore.py", line 331, in _leap_message_to_leap_mail
    body = yield self._raw_message_body(message)
  File "/Users/kfleerko/.virtualenvs/pixelated-user-agent/lib/python2.7/site-packages/twisted/internet/defer.py", line 1274, in unwindGenerator
    return _inlineCallbacks(None, gen, Deferred())
--- <exception caught here> ---
  File "/Users/kfleerko/.virtualenvs/pixelated-user-agent/lib/python2.7/site-packages/twisted/internet/defer.py", line 1128, in _inlineCallbacks
    result = g.send(result)
  File "/Users/kfleerko/code/gh/pixelated/pixelated-user-agent/service/pixelated/adapter/mailstore/leap_mailstore.py", line 345, in _raw_message_body
    content_doc = (yield message.get_wrapper().get_body(self.soledad))
  File "/Users/kfleerko/.virtualenvs/pixelated-user-agent/src/leap.mail/src/leap/mail/adaptors/soledad.py", line 693, in get_body
    d = store.get_doc('C-' + body_phash)
exceptions.TypeError: cannot concatenate 'str' and 'NoneType' objects
```

**Comment by kaeff** [comment]
Not observed on wazokazi right now. -> retest before starting development

**Comment by kaeff** [comment]
Might be related to server load, `GET /mail` and `GET /mail/someid` sometimes return a 503 with the following body:

    Could not connect to instance kaeff2: HTTP 404: Not Found

**Comment by shyba** [comment]
Anyway... it seems that body_phash was None. I will double check that scenario as well.

**Comment by shyba** [comment]
As commented on #leap channel today, that might be a known issue.
https://leap.se/code/issues/7450
^ There are bugs related to sending mails to self: https://leap.se/code/issues/7264

We should keep an eye on both issues when seeing this kind of bug.

**Comment by jeffhsta** [comment]
The problem described on this issue is not happening any more, so I am closing this one
